pyat/test/test_lattice_object.py

```diff
@@ -41,62 +40,62 @@ def test_lattice_creation_from_lattice_inherits_attributes():
     assert lat._radiation is True
     assert lat.an_attr == 12
     assert lat.another_attr == 5
-
-
-def test_lattice_gets_attributes_from_RingParam():
-    rp = elements.RingParam('lattice_name', 3.e+6, Periodicity=32)
-    l = Lattice([rp])
-    assert len(l) == 0
-    assert l.name == 'lattice_name'
-    assert l.energy == 3.e+6
-    assert l.periodicity == 32
-    assert l._radiation is False
-    assert len(Lattice([rp], keep_all=True)) == 1
-
-
-def test_lattice_gets_attributes_from_elements():
-    d = elements.Dipole('d1', 1, BendingAngle=numpy.pi, Energy=3.e+6,
-                        PassMethod='BndMPoleSymplectic4RadPass')
-    l = Lattice([d])
-    assert len(l) == 1
-    assert l.name == ''
-    assert l.energy == 3.e+6
-    assert l.periodicity == 2
-    assert l._radiation is True
+#
```

Comment:
Commented tests: some should be deleted, others should run on the `_scanner()` function where these things are checked now?

Reply:
Commented tests are now deleted. For checking the internals of `load_mat`, there are two possibilities:

1. Ignoring the implementation and checking only the result as it is documented: for this, one would need a set of Matlab files with different configurations and checking the expected behaviour. This looks very heavy.
2. Checking the implementation by importing private functions: then the tests must be updated if the implementation changes. I'll try to restore some of the removed tests in that way